content/master/guides/extensions-release-process.md

```diff
@@ -0,0 +1,223 @@
+providers and functions and pushing them to an OCI registry such as `ghcr.io`.
+
+A typical GitHub workflow definition contains the following steps:
+
+1. Fetching the source repository
+2. Authenticating to a remote registry
+3. Building and packaging artifacts
+4. Pushing (publishing) the artifact
+
+{{< hint "warning" >}}
+The supplied credentials for the remote registry require read and write access
+as upload requests to the registry specify `push` authorization scope.
+{{< /hint >}}
+
+The template GitHub repositories for [providers] and [functions] provide
+a functional GitHub Action in `.github/workflows/ci.yml`. The following
+sections of this guide cover configuration options and conventions for each.
+
+## Common Configuration
+
+All workflows require references to credentials for a remote registry.
+Typically, users configure them as [GitHub Actions Secrets], and the workflow
+performs authentication via the`docker/login-action`
+[action](http://github.com/docker/login-action).
+
+For example, adding the following step to a pipeline authenticates
+the job to `ghcr.io` using the workflow's ephemeral GitHub OIDC token.
+
+```yaml
+      - name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+```
+
+{{< hint "important" >}}
+By default, the job's OIDC token don't have permission to write packages
+to `ghcr.io`. Permissions are configurable in the GitHub repository's settings
+or declared
+[explicitly](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token)
+in the workflow definition YAML file.
+{{< /hint >}}
+
+For other registries, it's still best practice to reference credentials as
+custom Secret variables. For example:
+
+```yaml
+      - name: Login to Upbound
+        uses: docker/login-action@v3
+        if: env.XPKG_ACCESS_ID != ''
+        with:
+          registry: xpkg.upbound.io
+          username: ${{ env.XPKG_ACCESS_ID }}
+          password: ${{ secrets.XPKG_TOKEN }}
+```
+
+## Branching conventions
+
+Repositories for Crossplane extensions follow similar branching conventions
+to upstream Crossplane, where the release process assumes the workflow
+executing in branches with the `release-*` prefix. `main` is often included,
+though a conventional release process would not build and push off of tags on
+`main`.
+
+```yaml
+on:
+  push:
+    branches:
+      - main
+      - release-*
+```
+
+For example, when releasing `v0.1.0` of an extension, the conventional
+process is to cut a release branch `release-0.1` at the git commit
+where it builds from, and tag it as `v0.1.0`.
+
+{{< hint "note" >}}
+Some custom workflows may accept an explicit input for the remote reference,
+which overrides inferring from a git ref or tag. The [`ci.yml`] file for
```

Comment:
this `ci.yml` link is not resolving/rendering on the preview site, e.g. in https://deploy-preview-913--crossplane.netlify.app/v2.0-preview/guides/extensions-release-process/#branching-conventions